Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8 AS builder
```

Comment:
I'd also add a rhel7 builder, which might be needed later on

i.e.

`FROM registry.ci.openshift.org/ocp/builder:rhel-7-golang-1.15-openshift-4.7 AS rhel7`

Reply:
I don't think it will. This component will only be deployed on OCP >= 4.8 which only runs rhel8.